src/features/ResourcesCard.tsx

```diff
@@ -99,4 +120,18 @@ const ResourcesCard: FC<{ categ: string; icon: IconDefinition }> = ({ categ, ico
 // Todo: Replace with actual empty state when ready in mockups.
 const EmptyState = () => <span>No resources available.</span>;
 
+const IconLookupCustom = iconName => {
```

Comment:
Same comment regarding moving this code to one spot..

Reply:
Done